app/controllers/AdminController.scala

```diff
@@ -669,4 +669,139 @@ class AdminController @Inject() (implicit val env: Environment[User, SessionAuth
       }
     )
   }
+
+  /**
+   * Gets street edge data for the coverage section of the admin page.
+   */
+  def getStreetEdgeData = UserAwareAction.async { implicit request =>
```

Comment:
Let's call this something more like "getCoverageData" (referencing the "Coverage" header on the Admin page). Let's change the name of the API endpoint accordingly